python/nav/activeipcollector/manager.py

```diff
@@ -45,7 +45,7 @@ def store(data):
 
     # Suspecting package drop - dividing into chunks and giving some
     # breathing room for each batch of updates.
-    chunks = [data[x:x+100] for x in range(0, len(data), 100)]
+    chunks = [data[x : x + 100] for x in range(0, len(data), 100)]
```

Comment:
PEP8 says: always the same amount of whitespace around colon in slices, but doesn't say anything about how much whitespace. I go with 0 so this always make my eyes bleed...

Reply:
You can have any color whitespace you like, as long as it's black.